src/MacVim/MMWindowController.m

```diff
@@ -1665,5 +1684,10 @@ - (void)enterNativeFullScreen
     [decoratedWindow realToggleFullScreen:self];
 }
 
+- (void)processAfterWindowPresentedQueue
+{
+    [[NSOperationQueue mainQueue] addOperations:afterWindowPresentedQueue waitUntilFinished:NO];
```

Comment:
Hmm, I don't find any good reason to use `NSOperationQueue` here. It would be on the main thread and we just need to execute blocks, right? How about

        for (void (^block)(void) in afterWindowPresentedQueue)
            block();

and

        [afterWindowPresentedQueue addObject:[block copy]];